app/views/shared/_footer.html.erb

```diff
@@ -1,15 +1,15 @@
 <footer>
   <section class="partners" id="partners">
     <ul>
-      <li><a href="https://www.vatlib.it/home.php" target="_blank"><%= image_tag 'vatican.png', alt: 'Biblioteca Apostolica Vaticana' %></a></li>
-      <li><a href="http://library.columbia.edu/" target="_blank"><%= image_tag 'columbia.png', alt: 'Columbia University Library' %></a></li>
-      <li><a href="http://www.frick.org/research/library" target="_blank"><%= image_tag 'frick.png', alt: 'Frick Art Reference Library' %></a></li>
-      <li><a href="https://www.getty.edu/research/" target="_blank"><%= image_tag 'getty.png', alt: 'Getty Research Institute' %></a></li>
-      <li><a href="http://library.harvard.edu/" target="_blank"><%= image_tag 'harvard.png', alt: 'Harvard University Library' %></a></li>
-      <li><a href="http://www.ub.uni-heidelberg.de/" target="_blank"><%= image_tag 'heidelberg.png', alt: 'Heidelberg University Library' %></a></li>
-      <li><a href="https://library.nga.gov/" target="_blank"><%= image_tag 'national.png', alt: 'National Gallery of Art Library' %></a></li>
-      <li><a href="http://library.princeton.edu/" target="_blank"><%= image_tag 'princeton.png', alt: 'Princeton University Library' %></a></li>
-      <li><a href="http://www.library.illinois.edu/" target="_blank"><%= image_tag 'illinois.png', alt: 'University of Illinois at Urbana-Champaign Library' %></a></li>
+      <li><a href="https://www.vatlib.it/home.php" target="_blank"><%= image_tag 'vatican.png', alt: 'Biblioteca Apostolica Vaticana', title: 'Biblioteca Apostolica Vaticana' %></a></li>
+      <li><a href="http://library.columbia.edu/" target="_blank"><%= image_tag 'columbia.png', alt: 'Columbia University Library', title: 'Columbia University Library' %></a></li>
+      <li><a href="http://www.frick.org/research/library" target="_blank"><%= image_tag 'frick.png', alt: 'Frick Art Reference Library', title: 'Frick Art Reference Library' %></a></li>
+      <li><a href="https://www.getty.edu/research/" target="_blank"><%= image_tag 'getty.png', alt: 'Getty Research Institute', title: 'Getty Research Institute' %></a></li>
+      <li><a href="http://library.harvard.edu/" target="_blank"><%= image_tag 'harvard.png', alt: 'Harvard University Library', title: 'Harvard University Library' %></a></li>
+      <li><a href="http://www.ub.uni-heidelberg.de/" target="_blank"><%= image_tag 'heidelberg.png', alt: 'Heidelberg University Library', title: 'Heidelberg University Library' %></a></li>
+      <li><a href="https://library.nga.gov/" target="_blank"><%= image_tag 'national.png', alt: 'National Gallery of Art Library', title: 'National Gallery of Art Library' %></a></li>
+      <li><a href="http://library.princeton.edu/" target="_blank"><%= image_tag 'pul-logo.png', alt: 'Princeton University Library', title: 'Princeton University Library' %></a></li>
```

Comment:
Adding the title attributes give users a popup with the institution's name on mouseover. Probably helpful since there are quite a few logos without any text.